core/common/src/ConcurrentQuery.ts

```diff
@@ -45,12 +45,19 @@ export interface QueryLimit {
 
 /** @public */
 export interface QueryPropertyMetaData {
+  /** The class name is set to empty if the property is a generated one, otherwise, it is the name of the ECClass that the property is contained within. */
   className: string;
+  /** Access string is the property's display label if the property is a generated one, otherwise it is the ECSQL property path. */
+  accessString?: string;
+  /** Indicates whether the property is a generated one or whether it directly refers to an existing ECProperty of one of the classes in the FROM or JOIN clauses. */
   generated: boolean;
   index: number;
   jsonName: string;
```

Comment:
We should document jsonName as the name of the property if the value was converted into json.  @khanaffan I'm not satisfied with my description ... got a better one?

Reply:
Updated the document, but 2nd opinion would be great @ColinKerr @khanaffan

Reply:
looks fine to me